package.json

```diff
@@ -65,12 +65,17 @@
     "test": "npm run cover"
   },
   "dependencies": {
+    "@google-cloud/automl": "^0.1.1",
```

Comment:
I don't think we want any of these dependencies in the root `package.json`.  These likely belong in `samples/package.json`.